corehq/apps/cloudcare/static/cloudcare/js/formplayer/constants.js

```diff
@@ -42,5 +42,7 @@ hqDefine("cloudcare/js/formplayer/constants", function () {
         BREADCRUMB_WIDTH_OFFSET_PX: 120.41, // unavailable breadcrumb space i.e. padding, home and hamburger icons
 
         COLLAPSIBLE_TILE_MAX_HEIGHT: 150,
+
+        DURATION_BEFORE_SHOW_LOADING: 1000,
```

Comment:
This is in ms, yeah? Could be clearer as `MS` / `MILLIS_BEFORE_SHOW_LOADING` or similar, but I don't feel strongly about it.

Reply:
second that

Reply:
Agreed. It is milliseconds. I'll rename it to make it more apparent.

Reply:
This delay functionality will be handled in a separate PR now,  so this has been removed. https://github.com/dimagi/commcare-hq/pull/34158/commits/7114ec7e574e361b83da29fb77cadb887debca81